src/index.ts

```diff
@@ -9,14 +10,63 @@ export declare function markdown(message: string): void
 
 import * as child_process from "child_process"
 import { distanceInWords } from "date-fns"
-import * as fetch from "node-fetch"
-import * as semver from "semver"
+import fetch from "node-fetch"
+import semver from "semver"
+import { URL } from "url"
+import { promisify } from "util"
 
-import * as includesOriginal from "lodash.includes"
-const includes = includesOriginal as any
+import includesOriginal from "lodash.includes"
+
+import { getProxyAgentFromUri } from "./getProxyAgentFromUri"
+import { getRegistries, Registry } from "./getRegistries"
+import { getYarnConfig, YarnConfig } from "./getYarnConfig"
+
+type IncludesFn = <TValue>(collection: TValue[], value: TValue) => boolean
+
+const exec = promisify(child_process.exec)
+const includes: IncludesFn = includesOriginal
+
+const escapePackage = (name: string): string => name.replace(/@/, "&#64;")
+const linkToNPM = (name: string): string => `https://www.npmjs.com/package/${name}`
+
+const defaultRenderPackageLink = (name: string): string => `<a href='${linkToNPM(name)}'>\`${escapePackage(name)}\`</a>`
+
+const defaultRenderReadme = npm => `
```

Comment:
- [ ] Improve the typing of `defaultRenderReadme`.

Reply:
could probably just make it - `({ readme }) =>`